### PR TITLE
GIX-2189: Use ReceiveModalPo in ReceiveModal.spec

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Add the token symbol at the title of the ReceiveModal.

In this PR, I move the ReceiveModal.spec to use page objects.

# Changes

* Add TestIdWrapper to SelectAccountDropdown.svelte

# Tests

* New PO SelectAccountDropdownPo
* Add method to ReceiveModalPo
* Move ReceiveModal.spec to use the ReceiveModalPo.

# Todos

- [ ] Add entry to changelog (if necessary).

Not necessary.

### CODE REVIEW

**Comment by lmuntaner** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #4341** <a href="https://app.graphite.dev/github/pr/dfinity/nns-dapp/4341?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/dfinity/nns-dapp/4341?utm_source=stack-comment).

File: frontend/src/lib/components/accounts/SelectAccountDropdown.svelte [hunk 1]
```diff
@@ -50,23 +51,25 @@
...
-    testId="select-account-dropdown"
-  >
-    {#each selectableAccounts as { identifier, name } (identifier)}
-      <DropdownItem value={identifier}
-        >{name ?? $i18n.accounts.main}</DropdownItem
-      >
-    {/each}
-  </Dropdown>
-{/if}
+<TestIdWrapper testId="select-account-dropdown-component">
```
Comment (lmuntaner) [inline frontend/src/lib/components/accounts/SelectAccountDropdown.svelte:54]: Only added the TestIdWrapper

**Comment by lmuntaner** [comment]
@dskloetd please review

**Review by dskloetd — APPROVED** [review]

File: frontend/src/lib/components/accounts/SelectAccountDropdown.svelte (hunk 1 above)
[the remark this answers is not in the document]
Comment (lmuntaner) [reply frontend/src/lib/components/accounts/SelectAccountDropdown.svelte:54]: Ah, I will make a note to change it.

**Pull request merged**